src/main.js

```diff
@@ -422,8 +424,20 @@ const i18n = new VueI18n({
   messages
 })
 
+Vue.config.errorHandler = function(err) {
+
+  var errorMessage = process.env.NODE_ENV === 'production' ? i18n.t('msgSomethingWentWrong') : err.toString();
```

Comment:
We might want to consider a global logging solution to still log the details of these errors.